lr_face/data.py

```diff
@@ -342,93 +346,96 @@ def _get_path(cls, person: str, idx: int) -> str:
         """
         return os.path.join(cls.RESOURCE_FOLDER, f'{person}',
                             f'{person}_{idx:04}.jpg')
-    
-    
-    
-class SCDataset(Dataset):   
+
+
+class SCDataset(Dataset):
     RESOURCE_FOLDER = os.path.join('resources', 'SCface')
-    
+
     def __init__(self, imagetype: List[str]):
         self.imagetype = imagetype
-    
-    
+
     @property
     @cache
     def images(self) -> List[FaceImage]:
         data = []
-        
+
         for type in self.imagetype:
             if type == 'frontal':
-                folder = os.path.join(self.RESOURCE_FOLDER, 'mugshot_frontal_cropped_all')
+                folder = os.path.join(self.RESOURCE_FOLDER,
+                                      'mugshot_frontal_cropped_all')
                 for filename in os.listdir(folder):
-                    path = os.path.join(folder,filename)
+                    path = os.path.join(folder, filename)
                     identity = filename[0:3]
                     data.append(FaceImage(
                         path,
                         identity,
+                        yaw=0,
                         source=str(self),
                         meta={
                             'cropped': True,
-                            'pose' : str('frontal'),
                             'cam': None,
-                            'dist' : None                            
+                            'dist': None
                         }
-                    ))                   
-                
-                
+                    ))
+
             elif type == 'rotated':
-                folder = os.path.join(self.RESOURCE_FOLDER, 'mugshot_rotation_all')
-                
+                folder = os.path.join(self.RESOURCE_FOLDER,
+                                      'mugshot_rotation_all')
+
                 for filename in os.listdir(folder):
-                    path = os.path.join(folder,filename)                    
+                    path = os.path.join(folder, filename)
                     name, file_extension = os.path.splitext(filename)
                     identity = filename[0:3]
-                    pose = name[4:]
+                    # we ignore information on left/right, just take the angle
+                    # 0 is frontal, 4 is sideways (1,2,3 intermediate steps)
+                    if name[4:] == 'frontal':
+                        pose = 0
```

Comment:
waarom hier pose en hierboven yaw? en kan in type=='rotated'  alsnog een 'frontal' foto zitten?

Reply:
nog oude naamgeving - aangepast. ja, in rotated an frontal zitten